### ISSUE TITLE
Text selection inside links

### ISSUE THREAD

**Issue by unxed** [OtterBrowser/otter-browser#109]
Opera 12 has an ability to select text placed inside A tag. It would be nice to have the same option included in Otter Browser.

**Comment by Emdek** [comment]
Since we have User Script for some time already we could do it as bundled and enabled by default User Script.
But first we would need some JS code with proper license...

**Comment by pierreporte** [comment]
Maybe @resetnow proposed to do it in [his comment](https://github.com/OtterBrowser/otter-browser/issues/109#issuecomment-37163343). He seems to be active on Github so maybe he could do it?

**Comment by resetnow** [comment]
The script mentioned above (https://gist.github.com/Griever/5005165) works just fine for me, at least with QtWebEngine. It's also MIT licensed, is this license compatible with Otter?

**Comment by pierreporte** [comment]
According to the GNU project, the MIT license is a name given to either the X11 license and the Expat. Both are compatible with the GPL v3 that Otter uses.

**Comment by pierreporte** [comment]
If we are going to provide official user scripts, it may be a good idea to indicate which scripts are provided by default and are officially supported, and which are not.

**Comment by Emdek** [comment]
Right, somehow I remembered that there was only some totally unreadable JS code to solve that...
And now I don't even see link to that solution in this ticket, probably I've seen it only during early research.

**Comment by unxed** [comment]
http://pastebin.com/781MeCvX

this one?

**Comment by Emdek** [comment]
@unxed, no, I remember something obfuscated or at least minified, either someone gave me link outside this ticket or I've found it on the net.

**Comment by unxed** [comment]
https://github.com/eight04/select-text-inside-a-link-like-opera
?
https://greasyfork.org/ru/scripts/789-select-text-inside-a-link-like-opera/code
?

**Comment by Emdek** [comment]
@unxed, nope, and there is no point for looking it up now. ;-)

**Comment by resetnow** [comment]
A previous version of the extension mentioned above (which is now removed) was obfuscated.

**Comment by beastie1** [comment]
You're talking about the one [mentioned here](https://github.com/OtterBrowser/otter-browser/issues/109#issuecomment-37127714).

**Comment by Emdek** [comment]
@resetnow, @beastie1, yeah, probably that one.
